CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Release History
 
-## 1.16.0 (TBD)
+## 1.16.0 (2024-05-01)
 
 ### New Features
 
 - Support stored procedure register with packages given as Python modules.
 - Added support for structured type schema parsing.
 
+### Bug Fixes
+
+- Fixed a bug when inferring schema, single quotes are added to stage files already have single quotes.
```

Comment:
Fixed a bug when inferring schema, single quotes are added to stage files already have single quotes.
->
Fixed a bug that when inferring a schema, single quotes are added to stage files that already have single quotes.